src/composables/useTour.ts

```diff
@@ -0,0 +1,384 @@
+        prepareDOMPrevPage?: () => Promise<void>,
+        prepareDOMNextPage?: () => Promise<void>,
+        onBeforeMountedNextStep?: () => Promise<void>,
+        onBeforeMountedPrevStep?: () => Promise<void>,
+        onMountedStep?: (cbu: () => void) => void,
+    };
+
+    ui: {
+        // Elements that must have opacity to focus attention in other elements in screen
+        elementsWithOpacity?: string[], // array of selectors
+
+        // Elements that shouldn't allow interactivity
+        elementsWithoutInteractivity?: string[], // array of selectors
+
+        disabledNextStep?: boolean,
+    };
+}
+
+export type TourSteps = {
+    [x in TourStepIndex]: TourStep;
+};
+
+// TODO Remove me
+export function useFakeTx(): Transaction {
+    return {
+        transactionHash: '0x123',
+        format: 'nim',
+        timestamp: 1532739000,
+        sender: useAddressStore().activeAddress.value || '',
+        recipient: '0x123',
+        senderType: 'nim',
+        recipientType: 'nim',
+        blockHeight: 1,
+        blockHash: '0x123456789ABCDEF',
+        value: 100_000,
+        fee: 1,
+        feePerByte: 1,
+        validityStartHeight: 1,
+        network: 'testnet',
+        flags: 0,
+        data: {
+            raw: 'My awesome data',
+        },
+        proof: {
+            raw: 'ES256K',
+        },
+        size: 1,
+        valid: true,
+        state: 'confirmed',
+    };
+}
+export function useOnboardingTourSteps({ root }: SetupContext): TourSteps {
+    const sleep = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
+
+    const closeAccountOptionsModal = async () => {
+        const sidebar = (document.querySelector('.column-sidebar') as HTMLDivElement);
+        sidebar!.removeAttribute('data-non-interactable');
+        await root.$nextTick();
+
+        const closeBtn = (document.querySelector('.modal .close-button') as HTMLDivElement);
+        closeBtn.click();
+
+        await sleep(400);
+    };
+
+    const openAccountOptionsModal = async () => {
+        const account = document.querySelector('.sidebar .account-menu') as HTMLButtonElement;
+        account.click();
+        await sleep(500); // TODO Check this random value
+    };
+
+    const steps: TourSteps = {
+        [MobileOnboardingTourStep.FIRST_ADDRESS]: {
+            page: '/',
+            tooltip: {
+                target: '.address-list > .address-button .identicon img',
+                content: [
+                    'This is your first address, represented by your avatar.',
+                    'Tap it to open the address details.',
+                ],
+                params: {
+                    placement: 'bottom-start',
+                },
+            },
+            lifecycle: {
+                onMountedStep: (cbu: () => void) => {
+                    const addressButton = document.querySelector('.address-list > .address-button');
+
+                    // eslint-disable-next-line no-unused-expressions
```

Comment:
I had to disable this alert in different locations, do you know how to avoid it?